modules/virt-virtctl-commands.adoc

```diff
@@ -82,9 +88,6 @@ the specified port of the node.
 |`virtctl version`
 |Display the client and server version information.
 
-|`virtctl help`
-|Display a descriptive list of `virtctl` commands.
-
```

Comment:
I just wanted to double check that this was intentionally removed

Reply:
Yes, this was intentional. I removed it because it is the first command we mention, before the table.

See the [preview](http://file.tlv.redhat.com/apinnick/BZ2053705-force-vm-shutdown/virt/virt-using-the-cli-tools.html#virt-virtctl-commands_virt-using-the-cli-tools).